app/performance_platform/total_sent_notifications.py

```diff
@@ -0,0 +1,45 @@
+from datetime import datetime
+
+from app import performance_platform_client
+from app.dao.notifications_dao import get_total_sent_notifications_in_date_range
+from app.utils import (
+    get_london_midnight_in_utc,
+    get_midnight_for_day_before,
+    convert_utc_to_bst,
+)
+
+
+def send_total_notifications_sent_for_day_stats(date, channel, count, period):
+    payload = {
+        '_timestamp': convert_utc_to_bst(date).isoformat(),
+        'service': 'govuk-notify',
+        'channel': channel,
+        'count': count,
+        'dataType': 'notifications',
+        'period': period
+    }
+    performance_platform_client.add_id_to_payload(payload)
```

Comment:
`performance_platform_client.create_payload({'channel': ..., 'count': ..., 'dataType': ...})`

look into this and how it can be generic - remove perf platform specific stuff from this business logic file if possible